src/blog/models.py

```diff
@@ -10,6 +10,12 @@
 class BlogIndexPage(Page):
     intro = RichTextField(blank=True)
 
+    def get_context(self, request):
+        context = super().get_context(request)
+        blogpages = self.get_children().live().order_by('-first_published_at')
+        context['blogpages'] = blogpages
+        return context
+
```

Comment:
Hmm, are you sure this is correct? When I check out your branch and make some pages, I get some weird results. See also the publish dates in the screenshot, which seem to go from 2120 to 1997 to 2023 to 2024. It seems to not use the custom "date" field (see the BlogPage model below) to order.

![afbeelding](https://github.com/drikusroor/brazil-blog/assets/8208970/e8a98045-5dfa-4276-8eac-406146585004)

Reply:
I did some research and apparently you can order by custom field like this:

```suggestion
    def get_context(self, request):
        context = super().get_context(request)
        blogpages = self.get_children().live().order_by('-blogpage__date').specific()
        context['blogpages'] = blogpages
        return context
```